packages/components/app/styles/components/form/masked-input.scss

```diff
@@ -11,9 +11,12 @@ $hds-form-masked-input-button-size: 24px;
 
 .hds-form-masked-input {
   position: relative;
+  display: grid;
+  grid-template-columns: auto min-content;
   width: 100%;
 
   .hds-form-masked-input__control {
+    grid-area: 1 / 1 / 2 / 2;
```

Comment:
what do you think using `grid-template-areas` in the parent element, and use the human-readable names in the `grid-area` declarations? it makes everything so much easier to read (and interpret), at least for me

Reply:
Added a commit https://github.com/hashicorp/design-system/pull/1587/commits/aa496455523d19b7dc68fc355062b8a63e8eb7d3 to convert/simplify these to grid areas